src/orders/service/model/GetOrderFunctionDefinition.ts

```diff
@@ -0,0 +1,19 @@
+import { Order, OrdersQueryRequestByBusinessId, OrdersQueryRequestByChannel, OrdersQueryRequestByChannelOrderId }
+  from '../../../../src';
+
+type GetOrderByOrderIdFunctionDefinition =
+  (orderId: string) => Promise<Order>;
+type GetOrderByBusinessIdFunctionDefinition =
+  (ordersRequestByBusinessId: OrdersQueryRequestByBusinessId) => Promise<Order[]>;
+type GetOrderByChannelIdFunctionDefinition =
+  (ordersRequestByChannel: OrdersQueryRequestByChannel) => Promise<Order[]>;
+type GetOrderByChannelOrderIdFunctionDefinition =
+  (ordersRequestByChannelOrderId: OrdersQueryRequestByChannelOrderId) => Promise<Order[]>;
+
+type GetOrderFunctionDefinition =
+  GetOrderByOrderIdFunctionDefinition &
+  GetOrderByBusinessIdFunctionDefinition &
+  GetOrderByChannelIdFunctionDefinition &
+  GetOrderByChannelOrderIdFunctionDefinition;
+
+export default GetOrderFunctionDefinition;
```

Comment:
I'm not sure what this model is for but it does not appear to be used anywhere. Please remove it.